config/puma.rb

```diff
@@ -1,2 +1,12 @@
 threads ENV['PUMA_MIN_THREADS'].to_i, ENV['PUMA_MAX_THREADS'].to_i
 workers ENV['PUMA_NUM_WORKERS'].to_i
+
+# Invoke Puma.stats from the master process
+before_fork do
+  Thread.new do
+    loop do
+      puts Puma.stats
+      sleep 30
+    end
+  end
+end
```

Comment:
this could be pretty noisy, can we add an env var to guard this on? like `if ENV['LOG_PUMA_STATS']`

Reply:
well, you should see the rest of the logs on the cloud.gov side, but yes, that's a good idea.

```
   2021-09-22T16:34:17.97-0700 [APP/PROC/WEB/0] ERR 34.216.215.190, 127.0.0.1 - - [22/Sep/2021:23:34:17 +0000] "POST /AssureIDService/Document/8a20a199d89fe9422dda6bb4d634021a/Image?light=0&side=0 HTTP/1.1" 200 - 2.0017
   2021-09-22T16:34:17.97-0700 [RTR/4] OUT identity-fake-server.app.cloud.gov - [2021-09-22T23:34:15.973574249Z] "POST /AssureIDService/Document/8a20a199d89fe9422dda6bb4d634021a/Image?light=0&side=0 HTTP/1.1" 200 259073 0 "-" "Faraday v1.1.0" "127.0.0.1:42140" "10.10.2.5:61064" x_forwarded_for:"34.216.215.190, 127.0.0.1" x_forwarded_proto:"https" vcap_request_id:"47dffa7c-d091-4e64-45f9-5b25452a1b84" response_time:2.006154 gorouter_time:0.000162 app_id:"b0478727-18c8-4609-92d2-1d3da8887bb6" app_index:"0" instance_id:"fc7203e6-0abe-402f-7cd1-dc6a" x_cf_routererror:"-" x_b3_traceid:"d71b7f3103e87edf" x_b3_spanid:"d71b7f3103e87edf" x_b3_parentspanid:"-" b3:"d71b7f3103e87edf-d71b7f3103e87edf"
   2021-09-22T16:34:17.99-0700 [RTR/3] OUT identity-fake-server.app.cloud.gov - [2021-09-22T23:34:15.996013539Z] "POST /AssureIDService/Document/d8d592c759996975479235dc660f8370/Image?light=0&side=0 HTTP/1.1" 200 259073 0 "-" "Faraday v1.1.0" "127.0.0.1:26768" "10.10.2.5:61064" x_forwarded_for:"34.216.215.190, 127.0.0.1" x_forwarded_proto:"https" vcap_request_id:"b00caf2e-d890-4725-7eaa-c1d178ad3dd2" response_time:2.003560 gorouter_time:0.000149 app_id:"b0478727-18c8-4609-92d2-1d3da8887bb6" app_index:"0" instance_id:"fc7203e6-0abe-402f-7cd1-dc6a" x_cf_routererror:"-" x_b3_traceid:"62ed7a92fae3b6f8" x_b3_spanid:"62ed7a92fae3b6f8" x_b3_parentspanid:"-" b3:"62ed7a92fae3b6f8-62ed7a92fae3b6f8"
   2021-09-22T16:34:17.99-0700 [APP/PROC/WEB/0] ERR 34.216.215.190, 127.0.0.1 - - [22/Sep/2021:23:34:17 +0000] "POST /AssureIDService/Document/d8d592c759996975479235dc660f8370/Image?light=0&side=0 HTTP/1.1" 200 - 2.0018
   2021-09-22T16:34:18.04-0700 [RTR/3] OUT identity-fake-server.app.cloud.gov - [2021-09-22T23:34:16.039124587Z] "POST /AssureIDService/Document/48f3638632eb6734b251a6552d84fa8c/Image?light=0&side=0 HTTP/1.1" 200 259073 0 "-" "Faraday v1.1.0" "127.0.0.1:26778" "10.10.2.5:61064" x_forwarded_for:"34.216.215.190, 127.0.0.1" x_forwarded_proto:"https" vcap_request_id:"f02cdd40-70c6-48a7-5544-61744c4c39b7" response_time:2.003365 gorouter_time:0.000262 app_id:"b0478727-18c8-4609-92d2-1d3da8887bb6" app_index:"0" instance_id:"fc7203e6-0abe-402f-7cd1-dc6a" x_cf_routererror:"-" x_b3_traceid:"2ac53f5290b03641" x_b3_spanid:"2ac53f5290b03641" x_b3_parentspanid:"-" b3:"2ac53f5290b03641-2ac53f5290b03641"
```

This is all that's needed from the above
```
34.216.215.190, 127.0.0.1 - - [22/Sep/2021:23:34:17 +0000] "POST /AssureIDService/Document/8a20a199d89fe9422dda6bb4d634021a/Image?light=0&side=0 HTTP/1.1" 200 - 2.0017
34.216.215.190, 127.0.0.1 - - [22/Sep/2021:23:34:17 +0000] "POST /AssureIDService/Document/d8d592c759996975479235dc660f8370/Image?light=0&side=0 HTTP/1.1" 200 - 2.0018
```

Reply:
How about https://github.com/18F/identity-fake-server/pull/15/commits/9d66a3edf0a01b430cf5670772407c3e98c37670?